result.js

```diff
@@ -42,13 +42,32 @@ function checkCookie(nameOfCookie) {
     }
 }
 
+const resultDiv = document.getElementById("result");
 
+function fadeIn(el) {
+    el.classList.add('show');
```

Comment:
Good job on using classList and modifying it instead of assigning the classes directly :D